redbot/cogs/mod/mod.py

```diff
@@ -111,8 +109,15 @@ def cog_unload(self):
                 "Please use `[p]moveignoredchannels` if "
                 "you were previously using these functions."
             )
-            self.bot.loop.create_task(send_to_owners_with_prefix_replaced(self.bot, msg))
-            await self.settings.version.set(__version__)
+            self.bot.loop.create_task(self.bot.send_to_owners_with_prefix_replaced(msg))
```

Comment:
```suggestion
            self.bot.loop.create_task(self.bot.send_to_owners_with_prefix_replaced(msg))
            await self.settings.version.set("1.1.0")
```

Reply:
Not necessary to write to config twice in this scenario, the save to config was moved to a single operation outside the if statements so it's set once and easier to make changes i nthe future.

Reply:
In case there is some error during any migration, we do not want the migrations before it to run another time once you try loading the cog again.